src/openrct2/scripting/bindings/object/ScObject.hpp

```diff
@@ -348,7 +343,7 @@ namespace OpenRCT2::Scripting
             auto entry = GetEntry();
             if (entry != nullptr)
             {
-                return entry->restraint_image_id;
+                return entry->SpriteGroups[static_cast<uint8_t>(SpriteGroupType::RestraintAnimation)].imageId;
```

Comment:
```suggestion
                return entry->SpriteGroups[EnumValue(SpriteGroupType::RestraintAnimation)].imageId;
```

Also for all others below.